src/main/java/com/typee/storage/TypeeStorage.java

```diff
@@ -0,0 +1,13 @@
+package com.typee.storage;
+
+import com.typee.commons.exceptions.DataConversionException;
+import com.typee.model.Tab;
+
+import javafx.collections.ObservableList;
+
+/**
+ * Represents a storage for Typee Application
```

Comment:
Represents Storage for the Typee application.

Reply:
noted. will update in the next commit :)